presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataReader.java

```diff
@@ -391,6 +393,21 @@ private static DateStatistics toDateStatistics(HiveWriterVersion hiveWriterVersi
                 dateStatistics.hasMaximum() ? dateStatistics.getMaximum() : null);
     }
 
+    private static TimestampStatistics toTimestampStatistics(HiveWriterVersion hiveWriterVersion, OrcProto.TimestampStatistics timestampStatistics, boolean isRowGroup)
+    {
+        if (hiveWriterVersion == ORIGINAL && !isRowGroup) {
```

Comment:
Can you add a comment on why we are ignoring file and stripe timestamp statistics?